builder/builder-config/src/main/java/io/helidon/builder/config/spi/GeneratedConfigBean.java

```diff
@@ -28,17 +32,25 @@ public interface GeneratedConfigBean extends GeneratedConfigBeanCommon {
  */
 
     /**
-     * Set the instance id of this config bean.
+     * Returns the instance id assigned to this bean.
+     * Note that the instance id is typically assigned the {@link Config#key()} in most circumstances.
      *
-     * @param val the new instance identifier
+     * @return the instance id assigned to this bean
      */
-    void __instanceId(String val);
```

Comment:
note: The setter was taken out of this interface and pushed down to the abstract impl.